tests/bootstrap.php

```diff
@@ -32,6 +32,39 @@
 	$_test_root = '/tmp/wordpress-tests-lib';
 }
 
+$required_constants = array(
+	'WORDPRESS_DB_PASSWORD',
+	'WORDPRESS_DB_HOST',
+	'WORDPRESS_DB_USER',
+	'WORDPRESS_DB_NAME',
+	'PHP_VERSION',
+	'COMPOSER_BINARY',
+	'WP_TESTS_DIR',
+	'WP_PHPUNIT__DIR',
+);
```

Comment:
Are these env vars or constants defined at this point?

Reply:
These are env variables set by `wp-env` in `db-env.js` and used in `build-docker-compose-config.js`.
```
'WORDPRESS_DB_PASSWORD',
'WORDPRESS_DB_HOST',
'WORDPRESS_DB_USER',
'WORDPRESS_DB_NAME',
```

These are also env variables:
```
'PHP_VERSION',
'WP_TESTS_DIR',
```

`WP_PHPUNIT__DIR` is an env var that is set even on local when user runs `composer install`
> It looks like on local I have the `WP_PHPUNIT__DIR` environment variable being set by Composer.

Reply:
And what about the scenario when this bootstrap file will be executed in a non `wp-env` environment?

Reply:
In case of local environment it would show error like this:
```bash
➜ composer test:plugins
> phpunit '--verbose' '--testsuite' 'auto-sizes'
Please use 'npm run test-php' instead of running the composer test command directly.
If you are intending to run PHPUnit directly, the following environment variables are missing: WORDPRESS_DB_PASSWORD, WORDPRESS_DB_HOST, WORDPRESS_DB_USER, WORDPRESS_DB_NAME, PHP_VERSION, WP_TESTS_DIR
Script phpunit handling the test event returned with error code 1
Script @test --verbose --testsuite auto-sizes was called via test:auto-sizes
Script @test:auto-sizes was called via test:plugins
```

Reply:
This message is irrelevant if someone tries to run tests from their custom setup. Also if the test setup is non-docker based, then I doubt if these globals/constants would ever be defined at this place in the bootstrap file.